xbmc/Application.cpp

```diff
@@ -434,7 +434,7 @@ bool CApplication::OnEvent(XBMC_Event& newEvent)
         actionId = newEvent.touch.action;
       else
       {
-        int iWin = g_application.GetActiveWindowID();
+        int iWin = g_windowManager.GetActiveWindowID();
         CButtonTranslator::GetInstance().TranslateTouchAction(iWin, newEvent.touch.action, newEvent.touch.pointers, actionId);
```

Comment:
right, could ofc remove iWin and wrap the line depending on prefered style. I assume the compiler optimizes away the temp variable so it shouldn't have any technical effect.